### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by deejayres**
create user show page and update navbar

### CODE REVIEW

File: app/assets/javascripts/users.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (brittanyjspringer) [inline app/assets/javascripts/users.coffee]: hi @deejayres please delete this file.

File: app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Users controller here.
```
Comment (brittanyjspringer) [inline app/assets/stylesheets/users.scss]: Please delete this one too.

File: app/helpers/users_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module UsersHelper
```
Comment (brittanyjspringer) [inline app/helpers/users_helper.rb]: Another file that can be removed!

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<h1>Welcome <%= @user.email %>!</h1>
+<p>Your to-do list will be here some day!</p>
```
Comment (brittanyjspringer) [inline app/views/users/show.html.erb:2]: Love this.

**Comment by brittanyjspringer** [comment]
@deejayres left some comments for you. Once these are resolved, please tag me in a PR comment here and I'll be happy to merge. Thanks!

**Comment by deejayres** [comment]
@Brit200313 okay, RM'd!

**Comment by brittanyjspringer** [comment]
popkey

**Pull request merged**